docs/conf/cincinnati/2018/news/welcome.rst

```diff
@@ -0,0 +1,63 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 18, 2018
+   :tags: cincinnati-2018, website, cfp, tickets, open-help
+
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
+
+Since 2011, the Open Help Conference in Cincinnati featured attendee-led discussions and sprints for a wide range of people working on open-source and community-based projects. This year, Write the Docs and Open Help are coming together to bring 5 days of talks, conversations, and doc sprints a little closer to home for our East Coast and Midwestern documentarians!
```

Comment:
👍

Reply:
I added a sentence at the end of the second paragraph to address @ericholscher 's "learn, discuss, do" comment:

"Write the Docs Cincinnati + Open Help 2018 creates a time and place for the community of documentarians to learn, discuss, and do, working together to improve the art and science of documentation."